### PR TITLE
Add simulation of Ca concentration against time

### PR DESCRIPTION
**Pull request by nmarquesin**
- user adds lake parameters
- user selects liming product or adds custom values
- user selects lenght of simulation
- plot of single product simulation calculation Ca concentration against time
- issue #11

### CODE REVIEW

File: subpages/lake_modelling.py [hunk 1]
```diff
@@ -1,12 +1,209 @@
...
+
+def getCustomLimParam():
+    st.markdown("#### Custom Liming parameters")
+    col1, col2 = st.columns(2)
+    lime_dose = col1.number_input('Liming "dose" (mg-lime/l)', min_value=0, value=10)
+    pct_ca = col1.number_input("Lime Ca content by mass (%)", min_value=0.0, value=38.5)
+    od_fac = col2.number_input("Overdosing factor (-)", min_value=0.0, value=1.5)
+    K_L = col2.number_input(
+        "Lime dissoltuion rate on the bottom of the lake (month^-1)",
+        min_value=1,
```
Comment (JamesSample) [inline subpages/lake_modelling.py:111]: Minor comment: the `min_value` for $K_L$ should be zero, not one.

**Review by JamesSample — APPROVED** [review]
This looks great - thanks Nathalie! Happy for this to be merged to 'main' so that people can play with it a bit.

**Review by IvanaH8 — APPROVED** [review]
Really nice code and page!

File: subpages/lake_modelling.py (hunk 1 above)
Comment (nmarquesin) [reply subpages/lake_modelling.py:111]: Fixed

**Pull request merged**